### ISSUE TITLE
can someone share the AP/AR/loss history of ResNet50 backbone?

### ISSUE THREAD

**Issue by Lyndon-wong** [yhenon/pytorch-retinanet#239]
I changed the backbone and train it, only get 2.9mAP with 10 epochs 's training. Can anyone share ResNet50 training history? Thanks!
